pingpong/panopto.py

```
@@ -0,0 +1,620 @@
+
+async def _panopto_legacy_login(access_token: str, tenant: str) -> dict[str, str]:
+    """Get session cookies via legacy login (required for SRT download)."""
+    panopto_config = get_panopto_config(tenant)
+    url = panopto_config.api_url("/auth/legacyLogin")
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(
+            url,
+            headers={"Authorization": f"Bearer {access_token}"},
+            allow_redirects=True,
+        ) as resp:
```

Ruff (0.15.6)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)